xdsl/dialects/x86/ops.py

```diff
@@ -2474,7 +2474,7 @@ class RM_VbroadcastsdOp(R_RM_Operation[AVXRegisterType, GeneralRegisterType]):
     name = "x86.rm.vbroadcastsd"
 
 
-class GetAnyRegisterOperation(Generic[R1InvT], IRDLOperation, X86Op):
+class GetAnyRegisterOperation(Generic[R1InvT], IRDLOperation, X86Op, ABC):
```

Comment:
is this by accident? if not, maybe we add this for this class in the `riscv` dialect too?

Reply:
I meant piggyback it in this PR too

Reply:
Oh yeah that would probably be a good idea.